devops/docker-compose.local.yml

```
@@ -9,16 +9,21 @@ networks:
 services:
 
   rabbitmq:
-    image: rabbitmq:3.12.6
+    image: rabbitmq:3-management
```

Should we expose the port to use the UI.
```suggestion
    image: rabbitmq:3-management
    ports:
      - 15672:15672
```